addons/snailmail/models/snailmail_letter.py

```diff
@@ -249,7 +258,14 @@ def _snailmail_print(self):
                     'note': note,
                     'user_id': letter.user_id.id,
                 }
-                self.env['mail.activity'].create(activity_data)
+
+                activity = letter.activity_id
```

Comment:
to self: on rebase - check that we indeed update the info partially to avoid email notifications when updating the activity